features/feed.feature

```diff
@@ -22,6 +22,7 @@ Feature: Feed
     Given a "Create(Note)" Activity "Note1" by "Alice"
     And "Alice" sends "Note1" to the Inbox
     And "Note1" is in our Inbox
+    And We waited for 1000 milliseconds
```

Comment:
Added this because the posts are now sorted by published_at and without the wait it's causing flakiness as the posts get published/created at the exact same time.